### PR TITLE
Some docs fixes + desktop -> tray

### PR DESCRIPTION
**Pull request by carsonfarmer**
Nothing serious here, just some minor typo fixes and docs updates and a move from desktop to tray.

### CODE REVIEW

**Comment by sanderpick** [comment]
Is the desktop rename in here?

**Comment by carsonfarmer** [comment]
ah the push didn't go thru? musta been a network issue. should be good now, just pushed again

**Comment by sanderpick** [comment]
small conflict ^

**Comment by sanderpick** [comment]
cool, GH has a nice conflict resolve UI now.

**Comment by sanderpick** [comment]
@carsonfarmer we probably need some changes to the CI config file, right?

**Comment by carsonfarmer** [comment]
Oh yup, good catch. Will push those updates in the AM.

**Comment by carsonfarmer** [comment]
Ok, you better take a look and hopefully approve this one before i keep adding stuff @sanderpick ;)

File: .circleci/config.yml [hunk 1]
```diff
@@ -227,7 +227,7 @@ jobs:
     - store_artifacts:
         path: ~/dist/cli
 
-  build-desktop-linux:
+  buil-tray-linux:
```
Comment (sanderpick) [inline .circleci/config.yml]: Small typo here but not a show stopper. Though, do we want to add these jobs to the workflow?

Comment (carsonfarmer) [reply .circleci/config.yml]: Was going with the etherem crowd... naw i'll fix that, and yes i'll add it to workflow 👍

**Pull request merged**